### DISCUSSION TITLE
OBS Midi not listnening to my ports

### DISCUSSION THREAD

**Discussion by briandsc** [cpyarger/obs-midi#143] [Q&A] [answered]
Hello. I have tried directly connecting my launchpad to obs midi and it did not recveive data

I tried connecting obs midi to ports from loop midi and it did not receive mapping data. 

I think perhaps I did something wrong. Can anyone advise? Win 10 user

Ok I was able to get it to listen. But. I cannot disable any devices. When I uncheck enable box the next time i open obs they are enabled again. 

Also, mapping works, but, i cant actually see the mappings in the config screen. I mapped Start record to note on but now in trying to map note off to stop record it um just starts recording when i "listen on" and hit the buitton i want. 

I am stuck

**Comment by cpyarger** [comment]
Please screen record what you are doing so I can see what may be wrong.

**Comment by cpyarger** [comment, marked as answer]
Biggest issues when it comes to it not listening are;

1. MIDI Device is being used by another application
2. People try manually setting midi values instead of using the ```listen one``` or ```listen many``` buttons

Disabling devices in the gui is having issues at the moment. You can go into your config file and change that manually. 

It may be that your button is set in ```Momentary``` mode. if so, it will automatically reset it's state when you release it. basically 'note on' on press. and 'note off' on release. I would check your devices manual and see about remapping your buttons to ```Toggle``` mode where it sends a 'note on' on the first press and a ' note off' on the second press
